Zapper-fi/studio
# feat: Implemented Honeyswap

## Description



## Checklist


## How to test?


Example address to check balance on Polygon: 0x62d7e2484fcfc6752b9473260efe1b86cabac34e
Example address to check balance on Gnosis: 0xe9812F14CdA5F02287A970F90e42c77EBB3cb6d7
